cfe_internal/enterprise/federation/federation.cf

```diff
@@ -399,12 +402,31 @@ bundle agent superhub_schema
       "$(cfengine_enterprise_federation:config.bin_dir)/psql_wrapper.sh"
         arglist => {
                      "cfdb",
-                     "'select superhub_schema();'",
+                     "\"select superhub_schema('$(sys.key_digest)');\"",
```

Comment:
Personally, I hate escaping quotes. In cfengine you can use backtickts, single quotes or double quotes as quoting characters. So, here where your value needs both single and double quotes, use backticks on the outside and you can avoid having to escape the inner double quotes.

Reply:
funny, you just told me that recently and I fixed the other one. Probably had this one "in the queue" and didn't think of fixing it also at the time. :thx: